harness.sh

```diff
@@ -754,7 +754,7 @@ function gabc_output_view_output {
 register gabc_output
 
 function tex_output_find {
-    $FIND tex-output -name '*.tex' -print0
+    $FIND tex-output -name '*.tex' ! -name 'gsp-sample.tex' -print0
```

Comment:
I think you might as well exclude `gsp-*.tex` here.  I don't see a problem with that.